src/psycopmlutils/timeseriesflattener/flattened_dataset.py

```diff
@@ -398,7 +408,9 @@ def flatten_temporal_values_to_df(
         pred_time_uuid_col_name: str,
         new_col_name: Optional[str],
         source_values_col_name: str = "value",
+        is_fallback_prop_warning_threshold: float = 0.9,
```

Comment:
The threshold here is based on a binary value with a prevalence of 1%: https://bit.ly/3LOGGBE